### PR TITLE
Write k3d logs to file

### PR DESCRIPTION
**Pull request by dennismeister93**
This PR adds writing to log file when working with runtime-k3d

### CODE REVIEW

File: runtime-local/src/run-service.py [hunk 1]
```diff
@@ -44,7 +44,11 @@ def run_specific_service(service_id: str) -> None:
             spinner.fail("💥")
             terminate_spawned_processes()
             print(f"Starting {service_id=} failed")
-            with open(get_log_file_name(service_id), mode="r", encoding="utf-8") as log:
+            with open(
+                get_log_file_name(service_id, "runtime-local"),
```
Comment (BjoernAtBosch) [inline runtime-local/src/run-service.py:44]: Shouldn't `get_log_file_name` be imported from velocitas_lib now?

File: velocitas_lib/velocitas_lib.py
Comment (BjoernAtBosch) [inline velocitas_lib/velocitas_lib.py:1]: General remark (not to be solved in this PR):
"velocitas_lib" sound quite "big" - shouldn't we rename it to "runtime_lib" or "common_lib"?

File: runtime-k3d/src/app_deployment/build_vehicleapp.py [hunk 2]
```diff
@@ -61,8 +68,8 @@ def build_vehicleapp():
                     ".",
                     "--no-cache",
                 ],
-                stdout=subprocess.DEVNULL,
-                stderr=subprocess.DEVNULL,
+                stdout=log_file,
+                stderr=log_file,
```
Comment (BjoernAtBosch) [inline runtime-k3d/src/app_deployment/build_vehicleapp.py]:
```suggestion
                stderr=subprocess.STDOUT,
                stdout=log_file,
```

**Review by MP91 — APPROVED** [review]
LGTM

**Pull request merged**